includes/playlist/playlist-functions.php

```diff
@@ -344,6 +344,14 @@ function mdjm_get_event_playlist_categories( $event_id, $args=array() )	{
 	return $categories;
 } // mdjm_get_event_playlist_categories
 
+/** Get the playlist limit for this event
+ *
+ */
+function mdjm_get_playlist_limit ( $event_id )  {
+     $playlist_limit = get_post_meta( $event_id, '_mdjm_event_playlist_limit', true );
+     return $playlist_limit;
```

Comment:
Let's allow filtering of this value in case there is ever a need for a developer or an add-on to override the setting...

`return apply_filters( 'mdjm_event_playlist_limit', $playlist_limit, $event_id );`

See [https://developer.wordpress.org/reference/functions/apply_filters/](https://developer.wordpress.org/reference/functions/apply_filters/)